WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.kt

```diff
@@ -283,21 +283,22 @@ class MySitesPage {
     }
 
     private fun scrollToCard(elementID: Int): MySitesPage {
-        WPSupportUtils.waitForElementToBeDisplayed(elementID)
         Espresso.onView(ViewMatchers.withId(elementID))
             .perform(scrollTo())
 
         return this
     }
 
-    fun scrollToDomainsCard(): MySitesPage {
-        return scrollToCard(R.id.dashboard_card_domain_cta)
-    }
-
```

Comment:
Just moved it a few lines below, so that it belongs to the same card-related functions.